packages/wrangler/src/index.tsx

```diff
@@ -515,12 +529,18 @@ export async function main(argv: string[]): Promise<void> {
 
       const envRootObj = args.env ? config.env[args.env] || {} : config;
 
+      if (!envRootObj.compatibility_date && !args["compatibility-date"]) {
+        console.warn(
+          "⚠️  Using the latest version of the Workers runtime. To silence this warning, please choose a specific version of the runtime with --compatibility-date, or add a compatibility_date to your wrangler.toml.\n"
+        );
+      }
+
```

Comment:
I agree with @petebacondarwin that the warning during dev is unnecessary. Removing it.